docs/node/config.md

```diff
@@ -30,7 +30,14 @@ minimum-gas-prices = "0.01133uluna,0.15uusd,0.104938usdr,169.77ukrw,428.571umnt,
 
 `app.toml`
 
-TODO: add details
+Enabling Rest API allows you to perform standard database functions (CRUD) using rest architecture.
+
+### Rosetta
+
+`app.toml`
+
+Rosetta is a blockchain integration API for Coinbase.
```

Comment:
Please add more details about Rosetta

https://www.rosetta-api.org/

Reply:
1 or 2 paragraphs should suffice

1) What it is
2) what enabling it in your node does

We should also add a relevant section on how to run it (similar to LCD REST API page)